packages/admin-ui/src/date-picker/date-picker.state.ts

```diff
@@ -100,19 +99,16 @@ export const useDatePickerState = (props: DatePickerInitialState = {}) => {
   }, [autoFocus, dateFieldState.first])
 
   const invalid = useMemo(() => {
-    // const min = new Date(minDateValue)
-    // const max = new Date(maxDateValue)
-    // return value < min || value > max
+    const outOfBounds =
+      date < calendarState.minDateValue || date > calendarState.maxDateValue
 
-    return invalidProp
-  }, [invalidProp])
+    return invalidProp || outOfBounds
+  }, [invalidProp, date])
```

Comment:
Does it make sense to add `minDateValue` and `maxDateValue` here?

Reply:
yes, It does